Gems/Atom/Feature/Common/Code/Include/Atom/Feature/RayTracing/RayTracingFeatureProcessorInterface.h

```diff
@@ -223,6 +162,25 @@ namespace AZ::Render
         virtual void SetProceduralGeometryTypeBindlessBufferIndex(
             ProceduralGeometryTypeWeakHandle geometryTypeHandle, const AZStd::unordered_map<int, uint32_t>& bindlessBufferIndices) = 0;
 
+        //! Creates a mostly empty MeshInfo-Entry suitable for a Procedural Geometry hit shader. This is a convenience wrapper arround
+        //! a call to MeshFeatureProcessor->AcquireMeshInfoEntry(), which allocates both, a MeshInfo - entry and a FallbackPBRMaterial -
+        //! entry.
+        //! \return MeshInfoHandle the handle (index) of the newly created meshInfo-Entry.
+        virtual MeshInfoHandle CreateMeshInfoForProceduralGeometry() = 0;
```

Comment:
This function (and `SetMaterialForProceduralGeometry`/`SetMaterialParametersForProceduralGeometry`) seems weird: If the whole point of this PR is to move the material handling away from the RT-FeatureProcessor, then these functions shouldnt exist. The `DebugDrawSystemComponent` (which is the only code where these functions are used) should acquire the mesh info directly from the MeshFeatureProcessor instead.